server/tinylicious/package-lock.json

```diff
@@ -453,13 +453,31 @@
         "lodash": "^4.17.21"
       },
       "dependencies": {
+        "@fluidframework/common-utils": {
+          "version": "0.32.1",
```

Comment:
@anthony-murphy - If we're updating tinylicous to depend on prerelease common-utils (33.1000-xxxxx), do we need to also update it to use pre-release server deps so there's not two versions of common-utils in play?

Or should we revert the update in tinylicious altogether if it doesn't need the new common-utils bits?

Reply:
you need to do none or both